opa-audit-operator/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+name: gatekeeper-audit
+description: |
+  This charm deploys the Gatekeeper Controller Manager.
```

Comment:
This charm deploys Gatekeeper Audit, not Gatekeeper Controller Manager.

Reply:
fixed